### PR TITLE
[Strawberry] Rename project

### PR DESCRIPTION
**Pull request by ArcasCZ**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -28,11 +28,11 @@ services:
       - POSTGRES_PORT=${BACKUP_DB_PORT}
       - SCHEDULE=${BACKUP_SCHEDULE}
   bot:
-    image: ghcr.io/pumpkin-py/pumpkin-py:latest
+    image: ghcr.io/strawberry-py/strawberry-py:latest
     container_name: ${INSTANCE_NAME}-bot
     env_file: .env
     volumes:
-      - ./:/pumpkin-py/:z
+      - ./:/strawberry-py/:z
```
Comment (ArcasCZ) [inline docker-compose.yml:35]: @j-fiala Just a sanity check, but do you think this might break something?

**Comment by ArcasCZ** [comment]
Closed and re-opened to run checks with new popie.

File: docker-compose.yml (hunk 1 above)
Comment (j-fiala) [reply docker-compose.yml:35]: If the naming stays consistent in folder structure, then It should not make any problems. LGTM

**Review by j-fiala — APPROVED** [review]
Accepted. Nicely refined names.

**Review by j-fiala — APPROVED** [review]
Well done on the naming updates. Approved!

**Pull request merged**